.github/workflows/dotnet-build-and-test.yml

```diff
@@ -74,6 +74,7 @@ jobs:
             .
             .github
             dotnet
+            python
```

Comment:
We need the python directory, too, since that is where the devui source/assets are